test/ui/toggle_3ds_ui_tests.js

```diff
@@ -107,7 +107,24 @@ describe('The toggle 3D Secure page when 3D Secure is off', function () {
 
     let body = renderTemplate('3d_secure/index', templateData)
 
-    body.should.containSelector('#threeds-helper-text')
+    body.should.containSelector('#3ds-helper-text-worldpay')
+  })
+
+  it('should not display helper test if not worldpay', function () {
+    let templateData = {
```

Comment:
I think the best practice currently not to use UI tests anymore, but to use `supertest` as for example in https://github.com/alphagov/pay-selfservice/tree/3a9370c80b190c090e3bf4248b7fe84e35fb8eb7/test/unit/controller/edit_merchant_details_controller
Because with UI tests we were skipping the controller logic, etc.
May be verify with someone else if the new way will be too complicated